protos/v1alpha1/resource_mapping.proto

```diff
@@ -49,6 +49,18 @@ message Resource {
   // Required. The full resource name. Example:
   // `//spanner.googleapis.com/projects/p1/instances/i1/databases/d1`
   string name = 2;
+
+  // Optional. The representation for subresource in the format of a URL. 
+  // Subscope is the resource that beyonds cloud resource level.
+  // like spanner table, GCS object.
+  // Format: "hierarchy_string?qualifier_string"
+  // where hierarchy_string represents the hierarchy of the subresource to this parent
+  // in the format of parent/foo/child/bar.
+  // and qualifier_string represents the additional information of the subresource
+  // in the format of: key1=value1&key2=value2.
+  // The subscope string should only have lowercase.
```

Comment:
"should only have lowercase" - this doesn't match the examples. You obviously have upper cases